src/persist-client/src/operators/shard_source.rs

```diff
@@ -522,7 +517,7 @@ where
                         // the shared timely output buffer.
                         fetched_output.give(&fetched_cap, fetched).await;
                         completed_fetches_output
-                            .give(&completed_fetches_cap, leased_part.into_exchangeable_part())
+                            .give(&completed_fetches_cap, ())
```

Comment:
We don't actually need to send the `()` right? Don't we have a pattern somewhere for something like `Stream<!>`? (I can't seem to find it, but I swear this exists somewhere.)

Reply:
`Infallible`, I think! I'll get a version of this up.